src/sentry/backup/imports.py

```diff
@@ -441,8 +441,33 @@ def import_in_global_scope(
     Sentry instance, some behaviors in this scope are different from the others. In particular,
     superuser privileges are not sanitized. This method can be thought of as a "pure"
     backup/restore, simply serializing and deserializing a (partial) snapshot of the database state.
+    During the setup of a fresh Sentry instance, there are a couple of models that are automatically created.
```

Comment:
I would move this comment into the test (see my note below as to where). Here, maybe we could just change the language to be a bit stronger ("doing a global input WILL clear all your existing data!" or similar).

Good feature actually: create a `y/N`-style dialog that informs the user of this, and makes them proactively agree that they're about to wipe all of their data.

Reply:
done